server/settings/general.py

```diff
@@ -36,6 +61,10 @@ class GeneralSettingsModel(BaseSettingsModel):
 
 DEFAULT_GENERAL_SETTINGS = {
     "add_self_publish_button": False,
+    "rop_output": {
+        "enabled": True,
+        "default_output_dir": "$HIP/ayon/"
```

Comment:
(cosmetics) 
we don't need the trailing `/` although keeping it give the feeling it's a directory. 
so, I think we should remove it when reading/getting the staging directory.
```suggestion
        "default_output_dir": "$HIP/ayon"
```

Reply:
https://github.com/ynput/ayon-houdini/pull/200/commits/1543271a0d3d6962977945d51df711160338e5d2